### ISSUE TITLE
Discourse external database configuration not working for me

### ISSUE THREAD

**Issue by ranzbak** [bitnami/charts#6606]
**Which chart**:
bitnami/discourse	4.0.0        	2.7.0      	A Helm chart for deploying Discourse to Kubernetes

**Describe the bug**
When an external database is configured, the resulting install is not able to connect to the external PostgreSQL database

**To Reproduce**

1. Create a Kubernetes cluster and a external host that runs the postgresql database
2. Create a values.yaml containing the external database configuration

```
  postgresql:
    enabled: false
  externalDatabase:
    host: rancher-test-13.it.esa.int
    user: discourse
    password: <passwd>
    postgresqlPostgresUser: <admin_user>
    postgresqlPostgresPassword: <password>
    database: 'discourse'
    port: 5432
```

3. Deploy using : helm install ask-it-esa-int -n acc-discourse -f values.yaml bitnami/discourse
4. When looking into the logs of pod : ask-it-esa-int-discourse
```
discourse 08:46:43.98
discourse 08:46:43.98 Welcome to the Bitnami discourse container
discourse 08:46:43.99 Subscribe to project updates by watching https://github.com/bitnami/bitnami-docker-discourse
discourse 08:46:43.99 Submit issues and feature requests at https://github.com/bitnami/bitnami-docker-discourse/issues
discourse 08:46:43.99
discourse 08:46:43.99 INFO ==> Validating settings in POSTGRESQL_CLIENT_* env vars
discourse 08:46:44.00 INFO ==> Trying to connect to the database server
discourse 08:47:44.18 ERROR ==> Could not connect to the database server
```

**Expected behavior**
Helm would configure Discourse in a way that it can connect.

**Version of Helm and Kubernetes**:
Kubernetes version: v1.19.9
Docker: 20.10.6
helm: 3.6.0

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.6.0", GitCommit:"7f2df6467771a75f5646b7f12afb408590ed1755", GitTreeState:"clean", GoVersion:"go1.16.3"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.1", GitCommit:"5e58841cce77d4bc13713ad2b91fa0d961e69192", GitTreeState:"clean", BuildDate:"2021-05-12T14:18:45Z", GoVersion:"go1.16.4", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.9", GitCommit:"9dd794e454ac32d97cde41ae10be801ae98f75df", GitTreeState:"clean", BuildDate:"2021-03-18T01:00:06Z", GoVersion:"go1.15.8", Compiler:"gc", Platform:"linux/amd64"}
WARNING: version difference between client (1.21) and server (1.19) exceeds the supported minor version skew of +/-1
```

**Additional context**
I created a shell in the container, which I used to to connect to the external database using:

psql -U discourse -h <database host name> -d discourse

Which gave me a database shell.
This excludes a firewall/DNS problem for me.

**Comment by marcosbc** [comment]
Hi @ranzbak, thanks for reporting this! You are indeed right, this stopped working in 4.0.0 after making some major changes to both the container image and chart.

I've submitted a PR to fix this issue in #6615. Once it is merged, our pipeline will start the release process for a new chart version which should be out later that day if everything goes fine.

**Comment by ranzbak** [comment]
Thank you very much!
